ci/install_geos.sh

```diff
@@ -36,13 +36,28 @@ build_geos(){
     pip install cmake
 
     echo "Building geos-$GEOS_VERSION"
+    rm -rf build
     mkdir build
     cd build
-     # Use Ninja on windows, otherwise, use the platform's default
+    # Use Ninja on Windows, otherwise, use the platform's default
     if [ "$RUNNER_OS" = "Windows" ]; then
         export CMAKE_GENERATOR=Ninja
     fi
-    cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=$GEOS_INSTALL -DCMAKE_INSTALL_LIBDIR=lib ..
+    # Avoid building tests, depends on version
```

Comment:
Was there a specific reason for this change, or just to minimize the build time?

Reply:
It is to align with  https://github.com/shapely/shapely/pull/1802/commits/7d54239a3b5656c2fd04098f8bde37ef4e6d8531 which minimizes build time